.github/workflows/codecov.yml

```diff
@@ -14,7 +14,11 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
       - uses: taiki-e/install-action@cargo-llvm-cov
-      - run: cargo llvm-cov --all-features --workspace --lcov --output-path lcov.info
+      - shell: bash
+        run: |
+          LLVMCOV_FEATURES=$(cargo metadata --format-version 1 --no-deps \
+            | jq -r '[ .packages[].features | keys[] | select(. != "default" and . != "aws-lc-rs-fips" and . != "rustls-aws-lc-rs-fips") ] | unique | join(",")')
```

Comment:
Can we just get the flattened version of this, without all the shell/jq magic? That stuff is unfamiliar (at least to me) and thus harder to maintain, and I don't think we change our features all that often.

Reply:
Happy to do so. I could also make it so that `jq` only minimally does `.packages[].features | keys[]`, and the rest is done using `grep -v`, `sort -u` and `paste -sd` if that's clearer.

Let me know what you decide.

Reply:
I think all of that is still more complexity than is warranted here.